megamek/src/megamek/server/totalWarfare/TWDamageManagerModular.java

```diff
@@ -731,6 +731,9 @@ public void damageMek(Vector<Report> reportVec, Mek mek, HitData hit, int damage
             // Apply damage to armor
             damage = applyEntityArmorDamage(mek, hit, damage, ammoExplosion, damageIS, areaSatArty, reportVec, mods);
 
+            // Apply playtest ammo explosion cap
+            damage = applyPlaytestExplosionReduction(mek, hit, damage, ammoExplosion, reportVec);
```

Comment:
Make this call conditional on `game.getOptions().booleanOption(OptionsConstants.PLAYTEST_1`, or rename the function.
This will make the function easier to unit test, plus it implicitly depends on that option so best to make the dependency explicit.